.pipeline/blubber.yaml

```diff
@@ -0,0 +1,23 @@
+version: v3
+base: docker-registry.wikimedia.org/nodejs-slim
+lives:
+  in: /srv/service
+runs:
+  environment: { APP_BASE_PATH: /srv/service }
+
+variants:
+  build:
+    base: docker-registry.wikimedia.org/nodejs-devel
+    apt: { packages: [git, build-essential, python-pkgconfig] }
+    node: {requirements: [package.json]}
+  test:
+    includes: [build]
+    entrypoint: [npm, test]
+  prep:
+    includes: [build]
+    node: { env: production }
+  production:
+    includes: [build]
```

Comment:
Doesn't need the `includes` if `copies` is present

Reply:
Yeah, I noticed that the `includes` line is omitted from the production variant in the docs as well, but when I remove it, all requests to the service in the resulting container throw 504s.

Reply:
hmmm this could be a bug then. Could you please file a bug and tag it with *release pipeline (blubber)*?